# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../gle-graphics/gcc41.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur gle-4.0.11-orig/src/gle/d_interface.h gle-4.0.11/src/gle/d_interface.h
--- gle-4.0.11-orig/src/gle/d_interface.h	2005-12-17 03:08:00.000000000 +0000
+++ gle-4.0.11/src/gle/d_interface.h	2006-03-31 09:54:58.099025500 +0000
@@ -219,7 +219,7 @@
 	void test_SVGFile();
 	void reapsfont(void);
 	string GetColor();
-	inline double SVGGLEDevice::AY(double y) { return Height-y; }
+	inline double AY(double y) { return Height-y; }
 };
 
 class GLEDummyDevice : public GLEDevice {
diff -Nur gle-4.0.11-orig/src/gle/sub.h gle-4.0.11/src/gle/sub.h
--- gle-4.0.11-orig/src/gle/sub.h	2005-12-17 03:08:00.000000000 +0000
+++ gle-4.0.11/src/gle/sub.h	2006-03-31 10:11:22.264532000 +0000
@@ -36,7 +36,7 @@
 	GLESubMap();
 	~GLESubMap();
 	void clear();
-	void GLESubMap::clear(int i);
+	void clear(int i);
 	GLESub* add(const string& name);
 	GLESub* get(const string& name);
 	void list();
